.github/workflows/push-image.yaml

```diff
@@ -0,0 +1,73 @@
+---
+name: Build and push image to ECR
+
+on:
+  workflow_dispatch:
+    inputs:
+      docker_base_distribution:
+        description: "Enter the Docker Base Distribution (bionic, focal, stretch, buster, or bullseye)"
+        required: true
+        type: string
+        default: bullseye
```

Comment:
Is this also needed? Looking at the [dockerfile](https://github.com/MinaFoundation/uptime-service-backend/blob/main/dockerfiles/Dockerfile-delegation-backend) it seems to be hardcoded to `FROM golang:1.18` (not sure what is the base image there, but it doesn't seem that this workflow is able to pick distribution based on this parameter)